packages/core/src/Stores/notification-store.js

```diff
@@ -0,0 +1,897 @@
+import React from 'react';
+import { WS } from 'Services';
+import { sortNotifications, sortNotificationsMobile } from '../App/Components/Elements/NotificationMessage/constants';
+import BaseStore from './base-store';
+import {
+    excluded_notifications,
+    getCashierValidations,
+    getStatusValidations,
+    hasMissingRequiredField,
+} from './Helpers/client-notifications';
+
+export default class NotificationStore extends BaseStore {
+    @observable is_notifications_visible = false;
+    @observable notifications = [];
+    @observable notification_messages = [];
+    @observable marked_notifications = [];
+    @observable push_notifications = [];
+    @observable client_notifications = {};
+    @observable should_show_popups = false;
+
+    constructor(root_store) {
+        super({ root_store });
+        reaction(
+            () => root_store.common.app_routing_history.map(i => i.pathname),
+            () => {
+                this.filterNotificationMessages();
+            }
+        );
+        reaction(
+            () => [
+                root_store.client.account_settings,
+                root_store.client.account_status,
+                root_store.client.landing_companies,
+                root_store.modules?.cashier?.general_store?.is_p2p_visible,
+                root_store.common?.selected_contract_type,
+                root_store.client.is_eu,
+            ],
+            () => {
+                if (
+                    root_store.client.landing_companies &&
+                    Object.keys(root_store.client.landing_companies).length > 0
+                ) {
+                    this.removeNotifications();
+                    this.removeAllNotificationMessages();
+                    this.handleClientNotifications();
+                }
+            }
+        );
+    }
+
+    @computed
+    get custom_notifications() {
+        const { has_malta_account, can_have_mlt_account, is_uk } = this.root_store.client;
+        const notification_content = {
+            mx_mlt_notification: {
+                header: () => {
+                    if (has_malta_account || can_have_mlt_account) {
+                        return localize('Your Options account is scheduled to be closed');
+                    } else if (is_uk) {
+                        return localize('Your Gaming account is scheduled to be closed');
+                    }
+                    return localize('Your account is scheduled to be closed');
+                },
+                main: () => {
+                    if (has_malta_account || can_have_mlt_account) {
+                        return localize('Withdraw all funds from your Options account.');
+                    } else if (is_uk) {
+                        return localize('Please withdraw all your funds as soon as possible.');
+                    }
+                    return localize('Please proceed to withdraw your funds before 30 November 2021.');
```

Comment:
Please check with someone for this, this date has already passed.

Reply:
@carol-binary oh, thank you so much!

Reply:
@carol-binary QA replied that the current decision is to leave it as it is